api/graphql/schema.graphql

```diff
@@ -577,6 +603,10 @@ type PoolCandidate {
   priorityVerificationExpiry: Date
     @rename(attribute: "priority_verification_expiry")
     @canRoot(ability: "viewNotes")
+
+  # fields grouped by permission
+  viewNotes: PoolCandidateViewNotes @self @canRoot(ability: "viewNotes")
+  viewStatus: PoolCandidateViewStatus @self @canRoot(ability: "viewStatus")
```

Comment:
Wasn't sure if we want to deprecate and replace the individual fields or not everywhere, like for when accessed from single candidate operations. 
It would be a big diff I imagine, so it can be a separate issue if desired

Reply:
As seen with the Storybook struggles, it can easily get trickier than expected 😛